WcaOnRails/app/views/competitions/_my_competitions_table.html.erb

```diff
@@ -1,11 +1,13 @@
 <% show_delegates ||= false %>
 <% registrations_by_competition_id ||= {} %>
 <% if competitions.length < 1 %>
-  <%= alert :info do %>
-    <% if !past %>
-      <%= t '.no_upcoming_competitions_html', link: link_to(t('.competitions_list'), competitions_path) %>
-    <% else %>
-      <%= t '.no_past_competitions' %>
+  <% if !bookmarked %>
```

Comment:
How does this look if the user does not have any bookmarked competitions? Should we also have a `no_bookmarked_competitions` message?

Reply:
Added a message:

![no_bookmarked_competitions](https://user-images.githubusercontent.com/3885929/63558005-0b8aa280-c519-11e9-835b-339bd9817e84.png)